core/Updates/3.13.6-b1.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins;
+
+use Piwik\Config;
+use Piwik\Updater;
+use Piwik\Updates as PiwikUpdates;
+
+/**
+ * Update for version 3.13.6-b1.
+ */
+class Updates_3_13_6_b1 extends PiwikUpdates
+{
+
+    public function doUpdate(Updater $updater)
+    {
+        $config = Config::getInstance();
+        $tracker = $config->Tracker;
+        $tracker['enable_fingerprinting_limited_when_cookie_disabled'] = 0;
```

Comment:
could we do this only when  window_look_back_for_visitor > say 43200? (if  window_look_back_for_visitor is not custom, then no impact to users afaik)

Reply:
as mentioned we're creating new visits after midnight anyway... so this should probably not be important... 
unless `create_new_visit_after_midnight = 0`. Right now it looks like we can safely always apply the shorter fingerprint.